docs/docs/recipes/manage-users/README.md

```diff
@@ -4,8 +4,19 @@ sidebar_position: 6
 
 # 🧑‍🚀 Manage users
 
-TODO: briefly summary
+User management is the core feature for the admin staff.
 
-- Manage users on the Admin Console page
-- Manage users by Logto Core management API
-- Manage user data on the storage
+For example, you want to:
+
+- View the activities of a user whose phone number is "12345678901" to find out when their account has been stolen.
+- Correct the illegal avatar of a user who signed up with the email "pedophilia@exmaple.com".
+- Help a user whose `username` is "forgetful-man" reset the password that they forget.
+- Delete a user named "John Doe" who has violated the terms of service.
```

Comment:
This example seems too strict to our users.
Just showing that the admin can delete a specific user indexed by username is good to go.

Reply:
It's up to the developer.
@logto-io/eng any idea?

Reply:
agreed with @darcyYe, the scenarios sounds weird 1, 2, 4... why do we need "when their account has been stolen", "the illegal", "who has violated the terms of service". we don't care. we need to remain neutral.

Reply:
Updated. PTAL

Reply:
Using the scenarios provided by @wangsijie 

```markdown
For example:

- An end-user fails to sign in since they forget their username. According to a screenshot containing the user's avatar and full name, search by the full name, filter by the avatar, and finally, find their username.
- Find the newly registered users yesterday, view their profile, custom data, and social connections, analyze their behavior and make necessary return visits.
- An employee leaves your company. To protect the company's data security, clear the employee's social sign-in information, reset their password, or even delete their account directly.
- Help a user reset the password that they forget.
- ……
```

PTAL